src/components/TrackMaintaining.tsx

```diff
@@ -123,31 +71,41 @@ function Header({ trackId }: { trackId: TrackIdentifier }): JSX.Element {
   )
 }
 
-interface TrackViewProps {
-  trackId: TrackIdentifier
-  view: View
-  onShowExercise: (exercise: ExerciseIdentifier) => void
-  onHideExercise: () => void
+function SwitchActionableState(): JSX.Element {
+  const [current, onChange] = useActionableState()
+
+  const doToggle = useCallback(() => onChange((prev) => !prev), [onChange])
+
+  return (
+    <SwitchToggle
+      inActiveLabel="All"
+      activeLabel="Actionable"
+      onToggle={doToggle}
+      actionableOnly={current}
+    />
+  )
 }
 
-function TrackView({
-  trackId,
-  view,
-  onShowExercise,
-  onHideExercise,
-}: TrackViewProps): JSX.Element | null {
-  switch (view) {
-    case 'details': {
-      return <ExerciseDetails trackId={trackId} onHide={onHideExercise} />
-    }
-    case 'launch': {
-      return <LaunchList trackId={trackId} />
-    }
-    case 'tree': {
-      return <ExerciseTree trackId={trackId} />
-    }
-    default: {
-      return null
-    }
-  }
+function TrackView(): JSX.Element | null {
+  const match = useRouteMatch()
+
+  return (
+    <Switch>
+      <Route path={`${match.path}/details`} component={ExerciseDetails} />
+      <Route path={`${match.path}/launch`} component={LaunchList} />
+      <Route path={`${match.path}/tree`} component={ExerciseTree} />
+      <Redirect to={`${match.path}/launch`} />
+    </Switch>
+  )
+}
+
+function TrackMaintainingViewSelect(): JSX.Element {
+  const match = useRouteMatch()
+
+  return (
+    <div className="btn-group w-100">
+      <PageLink to={`${match.url}/launch`}>Launch</PageLink>
```

Comment:
`url` instead of `path`? You probably want the consistency?

Reply:
Yep. Found a small mistake in the page link bit too :)